extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -336,6 +336,9 @@ export class SubscriptionService {
           // Viewer authorization is redirected to use local platform instead.
           this.platformStore_.resolveEntitlement('local',
               /** @type {!./entitlement.Entitlement}*/ (entitlement));
+        }).catch(reason => {
+          this.platformStore_.reportPlatformFailure('local');
```

Comment:
/cc @prateekbh 
Is this enough? Or will the platform store still try to wait for other services to respond?

Reply:
To select the which entitlement is best suited to unlock the page, it'll wait for all services to be resolved/failed.

https://github.com/ampproject/amphtml/blob/master/extensions/amp-subscriptions/0.1/platform-store.js#L361

But as soon as local platform uses a fallback the document will be unlocked as we have one entitlement with GrantStatus: granted

Reply:
Is this true still for the viewer-auth mode? I'm seeing bunch of `doesViewerProvideAuth_` checks around.

Reply:
Oh yes, so in viewerAuthMode the only registered service is `local`

https://github.com/ampproject/amphtml/blob/master/extensions/amp-subscriptions/0.1/amp-subscriptions.js#L322